CHANGELOG.md

```diff
@@ -10,9 +10,12 @@ All notable changes to this project will be documented in this file.
 > ### API Breaking
 > N/A
 >
-> ### Enhancements
-> N/A
-> 
+
+### Enhancements
+ - New **String** extensions: [#300](https://github.com/SwifterSwift/SwifterSwift/pull/300) by [LucianoPAlmeida](https://github.com/LucianoPAlmeida).
```

Comment:
It would be better to move the `#300 by LucianoPalmeida` to the bottom of all the bullet points. (In case someone else adds more String extensions). I see some other CHANGELOG entries have this wrong format too. You can fix them up in this PR if you'd like 😅

Reply:
@SD10 Fixed :)